handlers/eth1Block.go

```diff
@@ -182,13 +188,18 @@ func GetExecutionBlockPageData(number uint64, limit int) (*types.Eth1BlockPageDa
 			}
 		}
 
+		var isContractInteraction types.ContractInteractionType
```

Comment:
**Minor:** The name `isContractInteraction` is misleading as it suggest a boolean value. This is more than true or false though, this holds an interaction type.
This applies to various other code parts too.

Similar to this, `GetAddressIsContractAtBlock` for example is kind of misleading too. A better name would be something like `GetAddressContractInteractionsAtBlock`.
This also applies to various other code parts too.

Reply:
The names of the methods were changed but we may also want to rename the returned variable name in that case.
E.g. it was
`txIsContractList, err := BigtableClient.GetAddressIsContractAtTransactions(transactions, idxs)`
now it is
`txIsContractList, err := BigtableClient.GetAddressContractInteractionsAtTransactions(transactions, idxs)`
with `txIsContractList` remaining the same.

@D13ce WDYT?

Reply:
Jeah, that should be consistent. We should change that.